service-rpc/if/TCLIService.thrift

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
I think this change wasn't intentional. Could you please revert?

Reply:
thank you for your feedback! but I think we definetly need more whitespaces...
I could move it to the README if you agree!

Reply:
Agreed. However, I still think a new-line takes up too much space on the display. Could you please replace with NARROW NO-BREAK SPACE (U+202F)?

Reply:
I don't think that would be enough ! I think we need more space!!!!
![gif](https://media.giphy.com/media/ygAaR0n5RsyAM/giphy.gif)

Reply:
How about adding this gif then? It _is_ space and also takes up a lot of space?

Reply:
Any idea on how we could add it to the thrift descriptor as well?

Reply:
we could use something like:
```
curl https://camo.githubusercontent.com/1255e5457d47d2c73543a87b90e5c66dae992582/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f7967416152306e35527379414d2f67697068792e676966 |
  gzip | bzip2 | xz | base64 | hexdump -C
```
but this will only solve the that it will be 7bit clean - any idea how we could make it into an RPC call?